swagger/swagger.js

```diff
@@ -127,4 +127,13 @@ module.exports = function(RED) {
         this.deprecated = n.deprecated;
     }
     RED.nodes.registerType("swagger-doc",SwaggerDoc);
+    
+    RED.httpAdmin.get('/swagger-ui/reqs/*', function(req, res){
```

Comment:
Need to verify this cannot be hacked by using paths like `../../../etc/passwd` to break out of the directory you are serving content from.